packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/duck.js

```diff
@@ -206,6 +211,22 @@ function detailLogItemToggle(state, { payload }) {
   return { ...state, detailStates };
 }
 
+function addHoverIndentGuideId(state, { payload }) {
+  const { spanID } = payload;
+  const newHoverIndentGuideIds = new Set(state.hoverIndentGuideIds);
+  newHoverIndentGuideIds.add(spanID);
+
+  return { ...state, hoverIndentGuideIds: newHoverIndentGuideIds };
```

Comment:
Is there a reason not to use the object property shorthand?

```js
function addHoverIndentGuideId(state, { payload }) {
   const { spanID } = payload;
   const hoverIndentGuideIds = new Set(state.hoverIndentGuideIds);
   hoverIndentGuideIds.add(spanID);

    return { ...state, hoverIndentGuideIds };
 }
```

Reply:
the lack of shorthand is only slightly more verbose but helps specify the lack of mutation of state.